packages/back-end/src/util/permission-constants.ts

```diff
@@ -31,7 +34,8 @@ export const GLOBAL_PERMISSIONS = [
   "manageTags",
   "manageApiKeys",
   "manageIntegrations",
-  "manageWebhooks",
+  "manageEventWebhooks",
+  "manageSDKWebhooks",
```

Comment:
This should be moved to `ENV_SCOPED_PERMISSIONS`?

Reply:
This file in back-end is not being used. I deleted it in the other PR.